Request/ParamConverter/DoctrineParamConverter.php

```diff
@@ -50,8 +58,16 @@ public function apply(Request $request, ParamConverter $configuration)
             $configuration->setIsOptional(true);
         }
 
-        // find by identifier?
-        if (false === $object = $this->find($class, $request, $options, $name)) {
+        $errorMessage = null;
+        if ($expr = $options['expr']) {
+            $object = $this->findViaExpression($class, $request, $expr, $options);
+
+            if (null === $object) {
+                $errorMessage = sprintf('The expression "%s" returned null', $expr);
```

Comment:
Yes, there are safe as they are not displayed anywhere (by default at least). That's also why I don't want to provide translation ou possible customization for them.